packages/manager/src/features/Linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingActionMenu.tsx

```diff
@@ -59,6 +59,14 @@ export const LinodeNetworkingActionMenu = (props: Props) => {
     ? 'Linodes must have at least one public IP'
     : undefined;
 
+  const getAriaLabel = (): string => {
+    if (ipAddress && 'address' in ipAddress) {
+      return `Action menu for IP Address ${ipAddress?.address}`;
```

Comment:
```suggestion
      return `Action menu for IP Address ${ipAddress.address}`;
```
Can we remove the optional chaining `?` here? `IPAddress` should be a string.

Reply:
Yes, we can remove it. I tried testing that as well it is working totally fine in both the cases.